ddtrace/debugging/_encoding.py

```diff
@@ -289,17 +290,82 @@ def _get_fields(obj):
         return {s: _safe_getattr(obj, s) for s in get_slots(obj)}
 
 
-def _captured_value_v2(value, level=1):
-    # type: (Any, int) -> Dict[str, Any]
+def _captured_value_v2(value, level=MAXLEVEL, maxlen=MAXLEN, maxsize=MAXSIZE, maxfields=MAXFIELDS):
+    # type: (Any, int, int, int, int) -> Dict[str, Any]
     _type = type(value)
+
+    if _type in BUILTIN_SIMPLE_TYPES:
```

Comment:
Maybe we want to check for None as well and set "isNull":true as in spec?

Reply:
Ah good catch! 👀 I'll make the change.